### ISSUE TITLE
Merge sw-composite into raqote

### ISSUE THREAD

**Issue by RazrFalcon** [jrmuizel/raqote#132]
Is there a reason to have `sw-composite` as a separate crate? It's tiny and doesn't that different from the raqote itself. Wouldn't it be easier to have them as a single crate?

**Comment by jrmuizel** [comment]
Yeah it would be easier. My goal was to mirror the cairo/pixman split with the hope that sw-composite would be useful outside of raqote.

**Comment by RazrFalcon** [comment]
Will you accept a pull request?
